src/views/shared/account/account.view.jsx

```diff
@@ -50,7 +50,7 @@ function Account({
           <p className={classes.tokenName}>{token.name}</p>
         )}
         <p className={classes.fiatBalance}>
-          {CurrencySymbol[preferredCurrency].symbol} {fiatBalance.toFixed(2)}
+          {CurrencySymbol[preferredCurrency].symbol} {fiatBalance}
```

Comment:
Why removing the rounding here? This value is not rounded anywhere else, right?

Reply:
We just render Account in two components `pending-deposit-list.view` where fiatBlance directly do `getTokenAmountInPreferredCurrency` and `account-list.view` where you can follow the lead of https://github.com/hermeznetwork/wallet-ui/blob/a08820a3c9c810ec49be0b0556e6112ecfb08d79/src/utils/accounts.ts#L65 and everything ends on `getAmountInPreferedCurrency` that has been updated to fix every number with two decimals,